core/src/Cabana_VerletList.hpp

```diff
@@ -213,22 +215,37 @@ struct VerletListBuilder
     // Cell stencil.
     LinkedCellStencil<PositionValueType> cell_stencil;
 
+    // Check to count or refill.
+    bool refill;
+    bool count;
+
+    // Maximum neighbors per particle
+    std::size_t max_n;
+
     // Constructor.
     VerletListBuilder( PositionSlice slice, const std::size_t begin,
                        const std::size_t end,
                        const PositionValueType neighborhood_radius,
                        const PositionValueType cell_size_ratio,
                        const PositionValueType grid_min[3],
-                       const PositionValueType grid_max[3] )
+                       const PositionValueType grid_max[3],
+                       const std::size_t max_neigh )
```

Comment:
Let's have a sensible default for this. Better yet, lets default to zero and if we get zero, fall back on the old implementation. That way if the user doesn't provide a value they get decent performance and if they do, they will know it needs to be big enough to get the performance improvement.

Reply:
That's a good point. I switched the default (in the VerletList constructor) to zero